sale_stock_invoice_lot_quantity/views/report_invoice.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="custom_stock_account_report_invoice_document"
+        inherit_id="stock_account.stock_account_report_invoice_document"
+    >
+        <xpath
+            expr="//table[@name='invoice_snln_table']/tbody/t/tr"
+            position="attributes"
+        >
+            <attribute name="style">display:none;</attribute>
+        </xpath>
+        <xpath expr="//table[@name='invoice_snln_table']/tbody/t/tr" position="before">
```

Comment:
Please add a comment to make the intention clear.  I doubt that most people understand why we are adding `span` tags to some fields.

Reply:
I added the comment.